### DISCUSSION TITLE
Incorrect column name on relation

### DISCUSSION THREAD

**Discussion by tsvetann** [mikro-orm/mikro-orm#5195] [Q&A] [answered]
I have the following property declaration:

```
@ManyToOne(() => User, { index: true })
 user_id!: User;
```

After migrating up the column in the database is user_id_id. Is this magic expected? I expected the column name to be user_id. Is it documented somewhere? Thx

**Comment by B4nan** [comment, marked as answer]
column name is not property name, there is no magic here, just [naming strategy](https://mikro-orm.io/docs/naming-strategy). your property should be called `user` as that is what it holds, `User` entity instance, not its ID.
